pkgs/development/libraries/tbb/default.nix

```diff
@@ -1,106 +1,9 @@
-{ lib
-, stdenv
-, fetchurl
-, fetchFromGitHub
-, fixDarwinDylibNames
-}:
+{ callPackage }:
 
-stdenv.mkDerivation rec {
-  pname = "tbb";
-  version = "2020.3";
+rec {
+  tbb_2020_3 = callPackage ./2020_3.nix { };
 
-  outputs = [ "out" "dev" ];
+  tbb_2021_8 = callPackage ./2021_8.nix { };
```

Comment:
I suggest putting whole package definition here without an indirection. That way it will also be easier to review what changed for `default.nix` version.